src/commands/fun/blindfoldedroosen.ts

```diff
@@ -0,0 +1,19 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `You've been challenged to a sword fight. However, your opponent, Professor Roosen, has decided to wear a blindfold. Feeling lucky?`;
+	aliases = ['br', 'blindfoldedroosen'];
```

Comment:
You don't need to include the actual command name in the aliases, it's recognized by default. If you want an alias other than s;br then maybe another, shortened version of the command name?